### PR TITLE
make ast object safe

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

File: crates/ra_syntax/src/ast.rs [hunk 1]
```diff
@@ -25,15 +25,24 @@ pub use self::{
...
     fn cast(syntax: SyntaxNode) -> Option<Self>
     where
         Self: Sized;
+
     fn syntax(&self) -> &SyntaxNode;
 }
 
+#[test]
+fn assert_ast_is_object_safe() {
+    fn _f1(x: &dyn AstNode) {}
```
Comment (lnicola) [inline crates/ra_syntax/src/ast.rs]: Some missing underscores here :-).

Comment (matklad) [reply crates/ra_syntax/src/ast.rs]: ah, forgot that it faild, thanks!

**Pull request merged**